src/components/elementView/elementTable.tsx

```diff
@@ -691,12 +760,32 @@ class ElementTable extends React.Component<
           cancelButtonText="Cancel"
           confirmButtonText="Delete"
           intent="danger"
-          isOpen={this.state.isDeleteOpen}
+          isOpen={this.state.openAlert === ElementTableOpenAlert.DELETE}
           onCancel={this.handleDeleteCancel}
           onConfirm={this.handleDelete}
         >
           <p>Are you sure you want to delete?</p>
         </Alert>
+        <Alert
+          cancelButtonText="Cancel"
+          confirmButtonText="Confirm"
+          intent="danger"
+          isOpen={this.state.openAlert === ElementTableOpenAlert.GRANT_ADMIN}
+          onCancel={this.handleGrantAdminCancel}
+          onConfirm={this.handleGrantAdmin}
```

Comment:
this nugget right here is why I created the `selected_userid` field, the method `handleGrantAdmin` needs to know which user was selected when this button was clicked. if there is a better way to do this that i'm missing that doesn't required the selected user/row field on state, I can change this!